front/src/modules/rollingStock/helpers/utils.ts

```diff
@@ -214,7 +307,9 @@ export const checkRollingStockFormValidity = (
         if (
           filteredCurve.max_efforts.length < 2 ||
           filteredCurve.speeds.length < 2 ||
-          new Set(filteredCurve.speeds).size !== filteredCurve.speeds.length
+          new Set(filteredCurve.speeds).size !== filteredCurve.speeds.length ||
+          filteredCurve.max_efforts.some((maxEffort) => maxEffort > 1000000) ||
+          filteredCurve.speeds.some((speed) => speed > kmhToMs(600))
```

Comment:
Maybe this can be extracted in a small function

Reply:
Done